pages/map.tsx

```diff
@@ -0,0 +1,154 @@
+  InputAdornment,
+  Card,
+  CardContent,
+  Typography,
+  CardActionArea,
+} from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+import Layout from 'components/Layout';
+import { PrismaClient } from '@prisma/client';
+import styles from '../styles/Map.module.css';
+
+const prisma = new PrismaClient();
+
+const Map = dynamic(() => import('../components/Map'), {
+  ssr: false,
+});
+
+type HomeProps = {
+  orgs: PublicOrganization[];
+};
+
+const Home: React.FC<HomeProps> = ({ orgs }) => {
+  const router = useRouter();
+
+  // This is to verify whether or not the current user has a proper session configured to see the page.
+  // Will be implemented in the next PR.
+  // const [session, loading] = useSession();
+
+  return (
+    <Layout>
+      <div className={styles.pageFlex}>
+        <TextField
+          id="outlined-size-small"
+          placeholder="Explore Organizations"
+          fullWidth
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <SearchIcon />
+              </InputAdornment>
+            ),
+          }}
+          variant="outlined"
+        />
+
+        <div className={styles.pageContent}>
+          <div className={styles.leftCol}>
+            <div className={styles.filters}>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>Keyword</InputLabel>
+                <Select label="Keyword">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>Keyword</InputLabel>
+                <Select label="Keyword">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+              <FormControl className={styles.filter} variant="outlined">
+                <InputLabel>More</InputLabel>
+                <Select label="More">
+                  <MenuItem value="">
+                    <em>None</em>
+                  </MenuItem>
+                  <MenuItem value={1}>One</MenuItem>
+                  <MenuItem value={2}>Two</MenuItem>
+                  <MenuItem value={3}>Three</MenuItem>
+                </Select>
+              </FormControl>
+            </div>
+
+            <div className={styles.cards}>
+              {orgs.length !== 0 ? (
+                orgs.map((org) => (
+                  <Card className={styles.card} key={org.id}>
```

Comment:
Ah. Just remembered that I was seeing weird console errors about duplicate keys in mapping. Just noting to myself to fix this when I come back to this branch. Please let me know if you see these errors on your end as well. (Why would using the id create duplicate errors?)

Reply:
i'm not getting a duplicate key error but i am getting a value undefined doesn't exist